core/src/main/java/org/hibernate/ogm/storedprocedure/impl/NoSQLProcedureOutputsImpl.java

```diff
@@ -55,13 +57,19 @@ public NoSQLProcedureOutputsImpl(NoSQLProcedureCallImpl procedureCall) {
 		this.sessionFactory = procedureCall.getSession().getFactory();
 		this.gridDialect = this.sessionFactory.getServiceRegistry().getService( StoredProcedureAwareGridDialect.class );
 		if ( gridDialect == null ) {
-			throw new UnsupportedOperationException( String.format(
-					"Your grid dialect do not supports stored procedures. You tried to call '%s' procedure",
-					procedureCall.getProcedureName()
-			) );
+			String dialectType = dialectType( sessionFactory );
+			throw new UnsupportedOperationException( String.format( "Grid dialect %s does not support server side procedures: procedure '%s' has been called",
```

Comment:
Why not use the *org.hibernate.ogm.util.impl.Log*?

Reply:
That's a good question,  can you create a separate JIRA and merge the rest?

I was in doubt about this because when something is not supported we usually don't use Log, but this should likely be an HIbernateException.

Reply:
Yes I do

Reply:
@DavideD I think what you did is consistent with the other places in the code. We do throw an `UnsupportedOperationException` in other places.

Reply:
Thank you @gsmet! @DavideD: So I do not have to create any issue ok?